docs/recipes/create-immortal-spore.md

```diff
@@ -1,15 +1,24 @@
 # Create immortal spores on-chain
 
-Immortal is a Spore Extension that provides the following features:
-- An immortal spore will live on-chain forever and cannot be destroyed 
-- Spore Extensions can only be added on a spore on the spore's creation
+## What is immortal
 
-Immortal is a initial extension that comes with the Spore Protocol, which is why we call it Spore Core Extension. 
+`Immortal` is a Spore Extension that offers the following rules to spores:
 
-If you want to create a spore with immortal extension enabled, you need to pass the `immortal` parameter to the props of the `createSpore` API when calling it:
+1. Immortal extension is enabled for a spore if the spore has a `immortal=true` in its `SporeData.contentType`
+2. An immortal spore lives on-chain forever, and cannot be destroyed under any circumstances
+
+To create a spore with the immortal extension enabled, you need to pass the `immortal` parameter to the props of the `createSpore` API when calling it. After successfully creating an immortal spore, any attempt by the owner to destroy it will result in a failed transaction due to verification by the `SporeType` script.
```

Comment:
Once an immortal spore is successfully created, any attempt from the owner to destroy it will fail due to the verification by the `SporeType` script.